src/nvim/options.lua

```diff
@@ -819,6 +819,13 @@ return {
       varname='p_fixeol',
       defaults={if_true={vi=true}}
     },
+    {
+      full_name='floatblend', abbreviation='fb',
```

Comment:
`winfloatblend` or even just `winblend` makes this discoverable by `:set win<tab>`. And `winblend` is analogous to `pumblend`.